src/main/java/net/sf/mzmine/modules/visualization/spectra/PlotMode.java

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
This file doesn't appear to be in the upstream project. https://github.com/mzmine/mzmine2/tree/master/src/main/java/net/sf/mzmine/modules/visualization/spectra